claims.go

```diff
@@ -63,6 +64,11 @@ func getClaim(logger log.Logger, allClaims map[string]interface{}, claim string)
 	//
 	// Or we fork and/or PR go-oidc.
 
+	switch v := val.(type) {
+	case float32:
+	case float64:
```

Comment:
This should be `case float32, float64:` on a single line to cover both types.

Reply:
Unfortunately, the combined case doesn't work because of limitation in the Go compiler for multiple types in a `switch case`: https://github.com/golang/go/issues/36616#issuecomment-575451270. 

I can use `fallthrough`? https://stackoverflow.com/a/40822747/9698467

Reply:
Ah, missed that detail. Thanks for the context! What you've got now looks good to me.